### ISSUE TITLE
Fix nasty scoping error

### ISSUE THREAD

**Issue by mjgpy3** [PolyglotSymposium/mappy#26]
In the repl
```
m> [identity :foo]
:foo
m> [identity]
:foo
```
The previous environment seems to be hanging around.

**Comment by mjgpy3** [comment]
May only be a repl error, but it's probably more.

**Comment by mjgpy3** [comment]
Might be nice to do #17 before this

**Comment by mjgpy3** [comment]
Seems resolved in <=0.1.2.1. #66 will make the following work more expectedly
```
m> [identity :a]
:a
m> [identity]
 - The name `a` is not defined
```
